cmd/release-controller/main.go

```diff
@@ -126,15 +123,12 @@ func main() {
 
 	flagset.StringVar(&opt.ListenAddr, "listen", opt.ListenAddr, "The address to serve release information on")
 
-	// these flags are from the test-infra go flagsets for bugzilla and github
-	// this is an intermediate PR to allow us to add the flags to the cmd arguments
-	// in the release-controller pods before they become required in a later PR
-	flagset.StringVar(&opt.githubEndpoint, "github-endpoint", github.DefaultAPIEndpoint, "GitHub's API endpoint (may differ for enterprise).")
-	flagset.StringVar(&opt.graphqlEndpoint, "github-graphql-endpoint", github.DefaultGraphQLEndpoint, "GitHub GraphQL API endpoint (may differ for enterprise).")
-	flagset.StringVar(&opt.tokenPath, "github-token-path", "", "Path to the file containing the GitHub OAuth secret.")
-	flagset.StringVar(&opt.deprecatedTokenFile, "github-token-file", "", "DEPRECATED: use -github-token-path instead.  -github-token-file may be removed anytime after 2019-01-01.")
-	flagset.StringVar(&opt.bugzillaEndpoint, "bugzilla-endpoint", "", "Bugzilla's API endpoint.")
-	flagset.StringVar(&opt.apiKeyPath, "bugzilla-api-key-path", "", "Path to the file containing the Bugzilla API key.")
+	flagset.StringVar(&opt.pluginConfig, "plugin-config", "/etc/plugins/plugins.yaml", "Path to plugin config file.")
```

Comment:
You may not add new flags to this component that default to locations not on everyone's laptop.  This must default empty and must be explicitly set.